### PR TITLE
Add about page

### PR DESCRIPTION
**Pull request by rubenmoya**

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -49,86 +57,10 @@ body {
...
-tr,
-ul,
-var,
-video {
-  border: 0;
-  margin: 0;
-  outline: 0;
-  padding: 0;
-}
-
```
Comment (Kikobeats) [inline src/styles/main.scss]: why this change?

File: src/styles/main.scss [hunk 2]
```diff
@@ -14,17 +12,27 @@ $line-height: 24px;
...
+  line-height: 2.25rem;
+}
+
+h3, h4, h5, h6 {
+  font-size: 1.25rem;
+  line-height: 1.625rem;
 }
 
 p {
-  line-height: $line-height;
```
Comment (Kikobeats) [inline src/styles/main.scss]: why this change?

File: gatsby-node.js [hunk 3]
```diff
@@ -105,6 +128,17 @@ exports.createPages = async ({ graphql, boundActionCreators }) => {
       }
     })
   })
+
+  // markdown pages
+  markdowns.map(({ fields }) => {
+    createPage({
+      path: fields.slug,
+      component: path.resolve(`./src/templates/page.js`),
```
Comment (Kikobeats) [inline gatsby-node.js]: extract template in a variable like

```
 const jobTemplate = path.resolve('./src/templates/job.js')
```

File: src/styles/main.scss (hunk 1 above)
Comment (rubenmoya) [reply src/styles/main.scss]: Because resetting everything to 0 was breaking up the markdown style in the parsed markdown pages

File: src/styles/main.scss (hunk 2 above)
Comment (rubenmoya) [reply src/styles/main.scss]: Somehow it wasn't being applied, like the `.scss` wasn't parsed

File: gatsby-node.js (hunk 3 above)
Comment (rubenmoya) [reply gatsby-node.js]: 👍

File: src/styles/main.scss (hunk 2 above)
Comment (Kikobeats) [reply src/styles/main.scss]: and use css vars?

File: src/styles/main.scss (hunk 1 above)
Comment (Kikobeats) [reply src/styles/main.scss]: wait, what? so we can't use normalize.css or similar? I don't understand

Comment (rubenmoya) [reply src/styles/main.scss]: We can use normalize, the thing is we were resetting EVERYTHING to padding/margin 0, so the lists for example were broken, normalize doesn't do that

File: src/styles/main.scss (hunk 2 above)
Comment (rubenmoya) [reply src/styles/main.scss]: I don't think it's necessary since we won't change that usually
